### PR TITLE
Prepping runtime for 0.3.0

### PR DESCRIPTION
**Pull request by berndruecker**
PR for https://github.com/camunda/connector-sdk/issues/228

### CODE REVIEW

**Review by igpetrov — DISMISSED** [review]
I am curious, why do we need separate flags for webhook and polling?

It feels like, when webhook is on and polling off, it won't work anyway.

**Comment by berndruecker** [comment]
The other way round - I would find it logical to enable polling for inbound connectors, but switch webhooks off (because I only want to have inbound JMS for example). This way the properties should keep stable

**Comment by igpetrov** [comment]
> The other way round - I would find it logical to enable polling for inbound connectors, but switch webhooks off (because I only want to have inbound JMS for example). This way the properties should keep stable

Understood, thank you

**Comment by berndruecker** [comment]
@tmetzke Ready to be merged. 
This reintroduces `inbound` to the runtime, but disable side effected by the default properties (polling, webhook, Tomcat). 
Adjusted readmes.

File: inbound/README.md [hunk 1]
```diff
@@ -121,7 +62,7 @@ zbctl --insecure deploy resource example/pull-request-notification-hmac-on.bpmn
 zbctl --insecure deploy resource example/broken.bpmn
 ```
 
-### Start a process
+###' Start a process
```
Comment (igpetrov) [inline inbound/README.md:65]:
```suggestion
### Start a process
```

**Review by igpetrov — APPROVED** [review]
LGTM

**Review by igpetrov — APPROVED** [review]

**Review by tmetzke — COMMENTED** [review]
Nice, thanks @berndruecker 👍 
Happy to merge the runtimes again like this 🙂

**Review by tmetzke — APPROVED** [review]

**Pull request merged**